Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +493,125 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         return [delete]
     }
 }
+
+class SyncedTabsTableViewCell: UIView {
```

Comment:
If this is not a `UITableViewCell` subclass, remove `TableViewCell` from the name.